README.md

```diff
@@ -1,2 +1,126 @@
+```
+
+Executing this command will copy the default vROps report JSON configuration to a user specified folder. 
+
+All report settings can then be configured via the JSON file.
+
+The following provides information of how to configure each schema within the report's JSON file.
+
+### Report
+The **Report** sub-schema provides configuration of the vSphere report information
+
+| Schema | Sub-Schema | Description |
+| ------ | ---------- | ----------- |
+| Report | Name | The name of the As Built Report
+| Report | Version | The report version
+| Report | Status | The report release status
+
+### Options
+The **Options** sub-schema allows certain options within the report to be toggled on or off
+
+| Schema | Sub-Schema | Setting | Description |
+| ------ | ---------- | ------- | ----------- |
+| Options | AuthSource | Local/Authentication Source Name | Set location of where to authenticate user running report.
+| Options | AlertFilter | String value | Used to report only on Alerts with specified value in name.
+
+
+### InfoLevel
+The **InfoLevel** sub-schema allows configuration of each section of the report at a granular level. The following sections can be set
+
+| Schema | Sub-Schema | Default Setting |
+| ------ | ---------- | --------------- |
+| InfoLevel | Authentication | 3
+| InfoLevel | Roles | 3
+| InfoLevel | Groups | 3
+| InfoLevel | Users | 3
+| InfoLevel | Adapters | 3
+| InfoLevel | RemoteCollectors | 3
+| InfoLevel | Alerts | 3
+| InfoLevel | Symptoms | 3
+| InfoLevel | SuperMetrics | 3
+| InfoLevel | ServiceStatus | 3
+| InfoLevel | CustomGroups | 3
+
+There are 6 levels (0-5) of detail granularity for each section as follows;
+
+| Setting | InfoLevel | Description |
+| ------- | ---- | ----------- |
+| 0 | Disabled | does not collect or display any information
+| 1 | Summary** | provides summarised information for a collection of objects
+| 2 | Informative | provides condensed, detailed information for a collection of objects
+| 3 | Detailed | provides detailed information for individual objects
+| 4 | Adv Detailed | provides detailed information for individual objects, as well as information for associated objects
+| 5 | Comprehensive | provides comprehensive information for individual objects, such as advanced configuration settings
+
+\*\* *future release*
+
+### Healthcheck
+The **Healthcheck** sub-schema is used to toggle health checks on or off. Currently none configured.
+
+## Examples 
+- Generate HTML & Word reports with Timestamp
+Generate a vROps As Built Report for vROps Server 'vrops-01.corp.local' using specified credentials. Export report to HTML & DOC formats. Use default report style. Append timestamp to report filename. Save reports to 'C:\Users\Tim\Documents'
+```powershell
+New-AsBuiltReport -Target 'vrops-01.corp.local' -Username 'admin' -Password 'VMware1!' -Report VMware.vROps -Format Html,Word -OutputPath 'C:\Users\Tim\Documents' -Timestamp
```

Comment:
Move `Report` parameter position to be first

Reply:
fixed